src/Integration.Vsix/SonarLintDaemon/DaemonAnalyzer.cs

```diff
@@ -140,7 +143,7 @@ private void MakeRequest()
             {
                 daemon.Ready -= HandleDaemonReady;
                 daemonInstaller.InstallationCompleted -= HandleInstallCompleted;
-                daemonAnalyzer.InvokeDaemon(path, charset, detectedLanguages, consumer, projectItem);
+                daemonAnalyzer.InvokeDaemon(path, charset, detectedLanguages, consumer, projectItem, CancellationToken.None);
```

Comment:
Why isn't the cancellation token passed through to this class?

Reply:
I didn't want to make further changes to DelayedRequest, and the daemon is not supposed to be affected by this change anyway. Do you think I should change it?
Right now DaemonAnalyzer receives the cancellationToken but doesn't pass it along to DelayedRequest.

Reply:
At the very least it's worth documenting that the daemon doesn't currently support cancellation.